include/sequence.h

```diff
@@ -1,134 +1,138 @@
+#define NA_BGM_CHASE 0x03                       // "Chase" (NA_BGM_CHASE)
+#define NA_BGM_MAJORAS_THEME 0x04               // "Majora's Theme" (NA_BGM_STALKID)
+#define NA_BGM_CLOCK_TOWER 0x05                 // "Clock Tower" (NA_BGM_CLOCK_TOWER)
+#define NA_BGM_STONE_TOWER_TEMPLE 0x06          // "Stone Tower Temple" - used for both Stone Tower and the Temple (NA_BGM_RUIN_DUNGEON_FRONT)
+#define NA_BGM_INV_STONE_TOWER_TEMPLE 0x07      // "Stone Tower Temple Upside-down" - used for both Inverted Stone Tower and the Temple (NA_BGM_RUIN_DUNGEON_REVERSE)
+#define NA_BGM_FAILURE_0 0x08                   // "Missed Event 1" (NA_BGM_FAILURE_0)
+#define NA_BGM_FAILURE_1 0x09                   // "Missed Event 2" - used for failing the Milk Run (NA_BGM_FAILURE_1)
+#define NA_BGM_HAPPY_MASK_SALESMAN 0x0A         // "Happy Mask Saleman's Theme" (NA_BGM_OMENYA)
+#define NA_BGM_SONG_OF_HEALING 0x0B             // "Song Of Healing" (NA_BGM_DEMO_CURE)
+#define NA_BGM_SWAMP_REGION 0x0C                // "Southern Swamp" - used for both Southern Swamp and Woodfall (NA_BGM_DAMP_AREA)
+#define NA_BGM_ALIEN_INVASION 0x0D              // "Ghost Attack" (NA_BGM_UFO)
+#define NA_BGM_SWAMP_CRUISE 0x0E                // "Boat Cruise" (NA_BGM_JUNGLE_CREWS)
+#define NA_BGM_SHARPS_CURSE 0x0F                // "Sharp's Curse" (NA_BGM_SHARP)
+#define NA_BGM_GREAT_BAY_REGION 0x10            // "Great Bay Coast" - used for Great Bay Coast, Zora Cape, and Pinnacle Rock (NA_BGM_SEA_AREA)
+#define NA_BGM_IKANA_REGION 0x11                // "Ikana Valley" (NA_BGM_IKANA_AREA)
+#define NA_BGM_DEKU_PALACE 0x12                 // "Deku Palace" (NA_BGM_DEKU_KING)
+#define NA_BGM_MOUNTAIN_REGION 0x13             // "Mountain Village" - used for Mountain Village, Snowhead, and Goron Village exterior (NA_BGM_SNOW_AREA)
+#define NA_BGM_PIRATES_FORTRESS 0x14            // "Pirates' Fortress" (NA_BGM_PIRATE_AREA)
+#define NA_BGM_CLOCK_TOWN_DAY_1 0x15            // "Clock Town, First Day" - Loaded from the main Clock Town seqeuence (sequence 0x1D) (NA_BGM_DUMMY_21)
+#define NA_BGM_CLOCK_TOWN_DAY_2 0x16            // "Clock Town, Second Day" - Loaded from the main Clock Town seqeuence (sequence 0x1D) (NA_BGM_DUMMY_22)
+#define NA_BGM_CLOCK_TOWN_DAY_3 0x17            // "Clock Town, Third Day" - Loaded from the main Clock Town seqeuence (sequence 0x1D) (NA_BGM_DUMMY_23)
+#define NA_BGM_FILE_SELECT 0x18                 // File Select (NA_BGM_FILE_SELECT)
+#define NA_BGM_CLEAR_EVENT 0x19                 // "Event Clear" (NA_BGM_EVENT_CLEAR)
+#define NA_BGM_ENEMY 0x1A                       // "Battle" (NA_BGM_ENEMY)
+#define NA_BGM_BOSS 0x1B                        // "Boss Battle" (NA_BGM_BOSS00)
+#define NA_BGM_WOODFALL_TEMPLE 0x1C             // "Woodfall Temple" (NA_BGM_JUNGLE_DUNGEON)
+#define NA_BGM_CLOCK_TOWN_MAIN_SEQUENCE 0x1D    // Plays the "morning theme" at the start of a day, then loads one of sequences 0x15, 0x16, or 0x17 (NA_BGM_MARKET)
```

Comment:
```suggestion
#define NA_BGM_CLOCK_TOWN_MAIN_SEQUENCE 0x1D    // Loads one of sequences 0x15, 0x16, or 0x17 (NA_BGM_MARKET). Will also play the "morning theme" if it is the start of a day
```
I think I would rephrase it this way. If you enter clocktown mid-day, this sequence will load the other sequences without playing the morning theme, it's not always played. Whether it's played or not is controlled by z_kankyo in calling either `Audio_PlayMorningSceneSequence` or `Audio_PlaySceneSequence` which will then write the correct io to this sequence

Reply:
Yeah, your rephrasing makes sense. The morning theme thing is just sort of an extension to what this sequence *really* does.